nearcore/src/config.rs

```diff
@@ -334,6 +334,33 @@ fn default_enable_rocksdb_statistics() -> bool {
     false
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct StoreConfig {
+    /// Maximum number of store files being opened simultaneously.
+    #[serde(default = "default_max_open_files")]
+    pub max_open_files: i32,
+    /// Cache size for ColState column.
+    #[serde(default = "default_col_state_cache_size")]
+    pub col_state_cache_size: usize,
+}
+
+impl Default for StoreConfig {
+    fn default() -> Self {
+        StoreConfig {
+            max_open_files: default_max_open_files(),
+            col_state_cache_size: default_col_state_cache_size(),
+        }
+    }
+}
+
+fn default_max_open_files() -> i32 {
+    10 * 1000
```

Comment:
let's also mention how we got those numbers (that we run such and such experiment on such and such date).

Will be useful in the future.